lib/matplotlib/table.py

```diff
@@ -653,7 +653,7 @@ def table(ax,
           cellLoc='right', colWidths=None,
           rowLabels=None, rowColours=None, rowLoc='left',
           colLabels=None, colColours=None, colLoc='center',
-          loc='bottom', bbox=None, edges='closed',
+          loc='bottom', bbox=None, edges='closed', fontsize=None,
```

Comment:
```suggestion
          loc='bottom', bbox=None, edges='closed', *, fontsize=None,
```
Let's not accept this as positional.

The alternative to making `fontsize` an explicit parameter and changing all related signatures would be to extract it from kwargs. That would leave all signatures literally unchanged.